app/AccountService.js

```diff
@@ -0,0 +1,29 @@
+/*-----------AccountService Class Singleton Implementation----------------------------------*/
+
+var AccountService = {
+  // array with already registered users
+  registeredUsers:[],
+  // AccountService Class Methods
+  signIn:function(email,password){
+    var user = this.returnUserIfExists(email);
+    if (user != null && (user.getPassword()==password)){
+      user.login();
+      return "Login Succesful";
```

Comment:
Return error codes, not strings. Depending on the context, other error messages might be shown, but the logic here is independent of the view context in which it is used